R/MixedModelsCommon.R

```diff
@@ -2192,12 +2191,18 @@
           family            = glmm_family,
           seed              = .getSeedJASP(options),
           model_fun         = "glmer"
-        )
+        ), error = function(e) e )
         
       }
       
     }
     
+  if (inherits(model, "error")) {
+    if (model$message == "Dropping columns failed to produce full column rank design matrix")
+      .quitAnalysis(gettext("The specified combination of factors does not produce an estimable model. The most likely reason for this issue is a factor / combination of factors leading to more levels than are estimable."))
```

Comment:
The specified combination of factors does not produce an estimable model. A factor or combination of factors resulted in more levels than the effective sample size.

Reply:
thanks, changed